cmd/cli/kubectl-kyverno/processor/policy_processor.go

```diff
@@ -212,10 +215,49 @@ func (p *PolicyProcessor) ApplyPoliciesOnResource() ([]engineapi.EngineResponse,
 		responses = append(responses, validateResponse)
 		resource = validateResponse.PatchedResource
 	}
+
 	restMapper, err := utils.GetRESTMapper(p.Client, !p.Cluster)
 	if err != nil {
 		return nil, err
 	}
+
+	// Mutate Admission Policies
+	var gvr schema.GroupVersionResource
+	mapping, err := restMapper.RESTMapping(gvk.GroupKind(), gvk.Version)
+	if err != nil {
+		log.Log.V(3).Info("skipping MAPs due to missing kind", "gvk", gvk)
+	} else {
```

Comment:
At this point, we don't really know if there are MAPs, I suggest checking first if there are MAPs similar to what we did for VAPs:
https://github.com/kyverno/kyverno/blob/0ec1222ef0cbb94e8e738bd32ef4916d1191938b/cmd/cli/kubectl-kyverno/processor/policy_processor.go#L261-L269